### PR TITLE
FIX - CPE UI outline is empty for some applications until a change is created

### PR DESCRIPTION
**Pull request by IvoSG**
Fix for [1441.](https://github.com/SAP/open-ux-tools/issues/1441)

-  call syncOutline in the initOutline because the attached event is not triggered for every case
(if the app is already loaded before the event is attached)

### CODE REVIEW

**Review by GDamyanov — CHANGES_REQUESTED** [review]
Please add changeset

**Review by GDamyanov — DISMISSED** [review]
Change is small and easy to understand
Manual testing ✅

File: .changeset/fifty-sloths-march.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
+'@sap-ux/preview-middleware': patch
+---
+
+call syncOutline in the initOutline because the attached event is not triggered for every case(if the app is already loaded before the event is attac
```
Comment (zdravko-georgiev) [inline .changeset/fifty-sloths-march.md:6]: Please change to "fix outline initialisation for the case when application is loaded, but outline is empty"

**Review by zdravko-georgiev — CHANGES_REQUESTED** [review]

File: .changeset/fifty-sloths-march.md (hunk 1 above)
Comment (IvoSG) [reply .changeset/fifty-sloths-march.md]: Done

**Review by zdravko-georgiev — APPROVED** [review]
Simple and straightforward change
Changeset is OK
Excellent test coverage
Manual testing ✅

**Review by nikmace — APPROVED** [review]
The code looks good.
Coverage is good.
Tested locally.

**Pull request merged**